File: Trabajo9-seleccion/main.js
function ordenamientoPorSeleccion(arr) {
    const n = arr.length;
    
    for (let i = 0; i < n - 1; i++) {
        let indiceMinimo = i;
        
        for (let j = i + 1; j < n; j++) {
            if (arr[j] < arr[indiceMinimo]) {
                indiceMinimo = j;
            }
        }
        
        if (indiceMinimo !== i) {
            [arr[i], arr[indiceMinimo]] = [arr[indiceMinimo], arr[i]];
        }
    }
}

// Ejemplo de uso
let arreglo = [64, 25, 12, 22, 11];
console.log("Arreglo original: " + arreglo.join(" "));

ordenamientoPorSeleccion(arreglo);

console.log("Arreglo ordenado: " + arreglo.join(" "));

File: Trabajo9-seleccion/main.py
def ordenamiento_por_seleccion(arr):
    n = len(arr)
    
    for i in range(n - 1):
        indice_minimo = i
        
        for j in range(i + 1, n):
            if arr[j] < arr[indice_minimo]:
                indice_minimo = j
        
        if indice_minimo != i:
            arr[i], arr[indice_minimo] = arr[indice_minimo], arr[i]

# Ejemplo de uso
arreglo = [64, 25, 12, 22, 11]
print("Arreglo original:", " ".join(str(x) for x in arreglo))

ordenamiento_por_seleccion(arreglo)

print("Arreglo ordenado:", " ".join(str(x) for x in arreglo))